src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -363,5 +363,51 @@ public async Task TestCredentialsCopyInAsyncContext()
                 server.Stop();
             }
         }
+
+        [Fact]
+        public void TestSmtpDeliveryFormatSevenBit()
+        {
+            string sendSubject = "", sendAsyncSubject = "";
+
+            //Run twice to get the results of synchronous and asynchronous methods
+            for (var i = 0; i < 2; i++)
+            { 
+                SmtpServer server = new SmtpServer();
+                server.SupportSmtpUTF8 = true;
+
+                SmtpClient client = new SmtpClient("localhost", server.EndPoint.Port);
+                client.DeliveryFormat = SmtpDeliveryFormat.SevenBit;
+
+                MailMessage msg = new MailMessage("foo@example.com", "bar@example.com", "Test \u6d4b\u8bd5 Contain \u5305\u542b UTF8", "hello \u9ad8\u575a\u679c");
+                msg.HeadersEncoding = msg.BodyEncoding = msg.SubjectEncoding = System.Text.Encoding.UTF8;
+
+                try
+                {
+                    Thread t = new Thread(server.Run);
+                    t.Start();
+
+                    if (i == 0)
+                    {
+                        client.Send(msg);
+                        sendSubject = server.Subject;
+                    }
+                    else
+                    {
+                        client.SendMailAsync(msg).Wait();
+                        sendAsyncSubject = server.Subject;
+                    }
+                }
+                finally
+                {
+                    server.Stop();
+                }
+            }
+
+            //Comparing the results of synchronous and asynchronous methods. Prefixes are used to distinguish the next test.
+            Assert.Equal("subject:" + sendSubject, "subject:" + sendAsyncSubject);
+
+            //Most important inspection
+            Assert.Equal("=?utf-8?B?VGVzdCDmtYvor5UgQ29udGFpbiDljIXlkKsgVVRGOA==?=", sendAsyncSubject);
```

Comment:
Make the Base64 encoded string a constant, and add comment to it.

The comment format in .NET Core is: "// This is a comment."